backend/routes/models.py

```
@@ -447,6 +492,22 @@ async def delete_model(model_name: str):
+        cm = custom_models.get_custom_model(custom_id)
+        if not cm:
+            raise HTTPException(status_code=404, detail=f"Custom model '{custom_id}' not found")
+        hf_repo_id = cm.get("hf_repo_id", "")
+        try:
+            cache_dir = hf_constants.HF_HUB_CACHE
+            repo_cache_dir = Path(cache_dir) / ("models--" + hf_repo_id.replace("/", "--"))
+            if repo_cache_dir.exists():
+                shutil.rmtree(repo_cache_dir)
+            return {"message": f"Custom model {model_name} cache deleted successfully"}
+        except Exception as e:
+            raise HTTPException(status_code=500, detail=f"Failed to delete custom model cache: {str(e)}") from e
```

Ruff (0.15.10)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)